### DISCUSSION TITLE
Вход через sso

### DISCUSSION THREAD

**Discussion by khpavel** [totumonline/totum-ru-issues-and-discussions#486] [Есть идея, а что если...]
В нашей компании используется ldap, для входа в разные приложения используется sso сервис на базе keycloak.

Возможно ли подключить sso при входе в totum?

**Comment by ttm-alexeypolunin** [comment]
Стоит в плане после лицензий: https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/480#discussioncomment-7753277
